src/components/ShipToCustomerModal.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal">
+          <ion-icon :icon="closeOutline" slot="icon-only" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Ship to customer") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <main>  
+      <ion-item>
+        <ion-label position="floating">{{ $t("First name") }}</ion-label>
```

Comment:
Let's use fixed labels

Reply:
Ok sir, I have replaced the floating position with a fixed position.